tests/unit/cli/builder/test_cli_builder.py

```diff
@@ -0,0 +1,365 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+import os
+
+import mmcv
+import pytest
+from mmcv.utils import Registry
+from torch import nn
+
+from otx.cli.builder.builder import (
+    Builder,
+    get_backbone_out_channels,
+    update_backbone_args,
+    update_channels,
+)
+from otx.cli.utils.importing import get_otx_root_path
+from otx.mpa.utils.config_utils import MPAConfig
+from tests.test_suite.e2e_test_system import e2e_pytest_unit
+
+
+class TestOTXCLIBuilder:
+    @e2e_pytest_unit
+    def test_builder_build_task_config(self, tmp_dir_path):
+        """
+        <b>Description:</b>
```

Comment:
Is it a limitation of our test CI so we have to write HTML tags for formatting?

Reply:
I know there is no problem with CI. Currently, I brought the code docstring format of `tests/unit/api` as it is, but this PR can be a starting point because there is no fixed thing, and OTX does not actually have a unit-test. Is there a format you recommend?

Reply:
To avoid the docstring of the HTML tags method, the docstring of the existing code (in otx) has been borrowed as it is. https://github.com/openvinotoolkit/training_extensions/pull/1519/commits/fc590a79fddc0d6c695a186f904762ae4aea5f92